_episodes/interviews/2025-05-27-accesskit-with-matt-campbell-and-arnold-loubriat.md

```diff
@@ -0,0 +1,56 @@
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+With [AccessKit](https://accesskit.dev/), [Matt Campbell](https://github.com/mwcampbell) and [Arnold Loubriat](https://github.com/DataTriny) [AccessKit](https://accesskit.dev/) took on the ambitious task of abstracting over
+the accessibility APIs of several target OS' to offer toolkit providers one unified way to make their UIs accessible across platforms.
+
+### Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+- [@0:10] - Prelude
+- [@1:11] - Introducing Matt and Arnold.
+- [@1:45] - "What are you working on?"
+- [@2:00] - "What does it offer?"
+- [@2:28] - "Who would use AccessKit?"
+- [@2:51] - AccessKit bindings in other languages besides Rust.
+- [@3:14] - On unifying different accessibility protocols and chromium's push model.
+- [@4:47] - AccessKit's API for toolkits.
+- [@6:58] - Some accessibility-tree concepts within AccessKit
+- [@8:47] - "Arnold, when did you get involved?"
+- [@4:47] - AccessKit's API for toolkits.
+- [@9:08] - AccessKit's origin - Arnold wrote 6000 lines of code to find Matt.
+- [@12:56] - Matt's history with Windows.
+- [@14:00] - How [Quorum](https://www.washington.edu/accesscomputing/quorum-programming-language-0) relates to AccessKit,
+- [@15:52] - The challenges Quorum UI development - and thus toolkit developers general face.
+- [@18:12] - Arnold and Matt on the necessity for back-end testing, Android, iOS - but also on Linux for AT-SPI2.
+- [@21:45] - On [Newton](https://blogs.gnome.org/a11y/2024/06/18/update-on-newton-the-wayland-native-accessibility-project/) and it's relation to the Chromium accessibility backend.
+- [@27:55] - Newton on other compositers other than mutter.
+- [@29:20] - [Politics mentioned in Newton origin document](https://gitlab.gnome.org/GNOME/at-spi2-core/-/blob/main/devel-docs/new-protocol.rst?ref_type=heads) (also) apply to Wayland purist versus Gnome D-Bus frictions.
+- [@31:03] - Wrapping up: where can people Arnold [on GitHub as 'DataTriny'](https://github.com/DataTriny), or [Arnold Loubriat on LinkedIn](https://www.linkedin.com/in/datatriny/) and [DataTriny on Mastodon](https://fosstodon.org/@DataTriny).
+- [@31:39] [Matt Campbell](https://github.com/mwcampbell) and [the AccessKit website](accesskit.dev).
+- [@31:53] [Luuk van der Duim](https://github.com/luukvanderduim), [Odilia screen reader](https://github.com/odilia-app/odilia) and [atspi](https://github.com/odilia-app/atspi).
```

Comment:
This isn't really a description of the chapter. Would try to re-phrase this concisely, and then make the links sub-bullets (see some of the other show notes for a reference).